### DISCUSSION TITLE
Streaming realtime data possible?

### DISCUSSION THREAD

**Discussion by arboeh** [dbuezas/lovelace-plotly-graph-card#353] [General]
Is streaming realtime data continously possible with this plugin? See https://plotly.com/javascript/streaming/

**Comment by dbuezas** [comment]
It's already implemented. Set `refresh_interval` to `auto` or just remove it completely.
See https://github.com/dbuezas/lovelace-plotly-graph-card?tab=readme-ov-file#refresh_interval
